vividus-plugin-web-app/src/main/java/org/vividus/selenium/BrowserWindowSize.java

```diff
@@ -16,31 +16,17 @@
 
 package org.vividus.selenium;
 
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
 import org.openqa.selenium.Dimension;
 
 public class BrowserWindowSize
 {
     private final int width;
     private final int height;
-    private final String sizeAsString;
 
-    public BrowserWindowSize(String sizeAsString)
+    public BrowserWindowSize(int width, int height)
```

Comment:
The class is similar to https://github.com/SeleniumHQ/selenium/blob/trunk/java/src/org/openqa/selenium/Dimension.java, we need to refactor code to use selenium's class.
If we are not going to solve it in the scope of this PR, I propose to create a ticket for this

Reply:
https://github.com/vividus-framework/vividus/issues/2365